spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/support/JdbcQueryLookupStrategy.java

```diff
@@ -172,45 +168,7 @@ public RepositoryQuery resolveQuery(Method method, RepositoryMetadata repository
 			throw new IllegalStateException(
 					String.format("Did neither find a NamedQuery nor an annotated query for method %s", method));
 		}
-
-		@SuppressWarnings("unchecked")
-		private class BeanFactoryRowMapperFactory implements AbstractJdbcQuery.RowMapperFactory {
```

Comment:
This class is removed completely, since this API was not public in the first place.